### ISSUE TITLE
Electronics shelf sleeve placement

### ISSUE THREAD

**Issue by menthol1979** [florianL21/LED-ClockShelf#23]
Hello ! 
First of all, many thanks for your work! It is absolutely marvelous and I decided to build one. 
However, I have a (stupid maybe) question. DIY Machines places the electronics shelf on top right position. 
Based on your diagram, since your design starts at lower left, where this module should be placed?
On position 0, position 15 or position 7?

Thank you!

**Comment by florianL21** [comment]
Hi,

I built 2 of these already and on one of them I made an electronics box that contains the ESP and the PSU and only have the 3 wires required for driving the LEDs running up to the left bottom corner. 
On the other I packed the ESP and other components quite tightly together and placed it within the walls of the shelf, also on the left bottom corner. 
I have to say that I personally prefer the first solution as the PSU will anyway have to be on the floor and the self contained box was easier to build. 
Ofc that now means if anything goes wrong I now have to open the electronics box that has mains voltage in it. 
So ultimately it's always a trade off and it will come down to what you prefer

**Comment by menthol1979** [comment]
Hi Florian!
Thank you for your answer! 
That is, by looking at the JPG image :
![Screenshot 2024-06-25 234147](https://github.com/florianL21/LED-ClockShelf/assets/70651211/e6fb3f52-1754-40ce-b53a-56232f60792b)
Shall I assume on position 0 ? I know that this can be customized later, but as a default reference to make it work on first run without any changes.
Thanks!

**Comment by florianL21** [comment]
Ah sorry, I misunderstood. Yes the red arrow coming in from the left side towards the segment 0 is where the signal enters the LED strips. 
Note that the segments do have a direction to them so you always have to watch out to wire from the output of one segment to the input of the next one.
Thus the "input" wire has to be wired to the input side of the LED strip of segment 0 on this picture

**Comment by menthol1979** [comment]
Excellent, so the electronics sleeve goes to 0! Will pick up my board tomorrow and immediately start assembling. Thanks for all the great work Florian!

**Comment by berg-maia** [comment]
There is a way to use the original strip direction segments, from the orginal project? i just wanted to use ESP32 without re-soldering all again

**Comment by Godzookee** [comment]
> There is a way to use the original strip direction segments, from the original project? i just wanted to use ESP32 without re-soldering all again

Yes you can there is an option in the confirm for the original setup.  This is what I did as I did not find this to way after I started the original project.

In the [library_diy-machines.json](https://github.com/florianL21/LED-ClockShelf/blob/master/lib/LED_clock/library_diy-machines.json)

**Comment by berg-maia** [comment]
Thanks Godzooke, and you know where goes the pin for the spot lights?
